### ISSUE TITLE
Can't get list of systems

### ISSUE THREAD

**Issue by KTibow** [bachya/simplisafe-python#40]
I'm trying out the example code:
```import asyncio
from aiohttp import ClientSession
from simplipy import API
async def main() -> None:
    """Create the aiohttp session and run."""
    async with ClientSession() as websession:
        simplisafe = API.login_via_credentials("<EMAIL>", "<PASSWORD>", websession)
        # Get a dict of systems with the system ID as the key:
        systems = await simplisafe.get_systems()
        # >>> {"1234abc": <simplipy.system.SystemV2 object at 0x10661e3c8>, ...}
asyncio.get_event_loop().run_until_complete(main())
```
I'm getting the error: 
```Traceback (most recent call last):
  File "C:\Users\kende_000\OneDrive\Documents\simplisafe-get-locations.py", line 47, in <module>
    asyncio.get_event_loop().run_until_complete(main())
  File "C:\Program Files\Python36\lib\asyncio\base_events.py", line 466, in run_until_complete
    return future.result()
  File "C:\Users\kende_000\OneDrive\Documents\simplisafe-get-locations.py", line 43, in main
    systems = await simplisafe.get_systems()
AttributeError: 'coroutine' object has no attribute 'get_systems'
```

**To Reproduce**
1. Run the code:
```import asyncio

from aiohttp import ClientSession

from simplipy import API


async def main() -> None:
    """Create the aiohttp session and run."""
    async with ClientSession() as websession:
        simplisafe = API.login_via_credentials("<EMAIL>", "<PASSWORD>", websession)

        # Get a dict of systems with the system ID as the key:
        systems = await simplisafe.get_systems()
        # >>> {"1234abc": <simplipy.system.SystemV2 object at 0x10661e3c8>, ...}


asyncio.get_event_loop().run_until_complete(main())
```

2. See the error once you adjust the credentials. I'm using Python 3.6.1.

**Comment by KTibow** [comment]
Will you possibly help me if I @mention you, @bachya ?

**Comment by bachya** [comment]
This is a documentation error and has been addressed with 9c2790f3b02f.

To fix, you need to `await` the `API.login_via_credentials` call:

```
simplisafe = await API.login_via_credentials("<EMAIL>", "<PASSWORD>", websession)
```
